### PR TITLE
Made corrections Makefile and travis to conform skycoin standards 

### PR DESCRIPTION
**Pull request by ayuryshev**
- https://github.com/skycoin/skycoin/wiki/Skycoin-project-Makefile-standards
- https://github.com/skycoin/skycoin/wiki/Skycoin-projects%27-Travis-CI-standards

Closes: #235

### CODE REVIEW

**Comment by ayuryshev** [comment]
```bash
$ make                                                                                                              20:05:35
check                          Run linters and tests
build                          Install dependencies, build apps and binaries. `go build` with ${OPTS} 
run                            Run skywire-node on host
stop                           Stop running skywire-node on host
clean                          Clean project: remove created binaries and apps
install                        Install `skywire-node`, `skywire-cli`, `manager-node`, `therealssh-cli`
lint                           Run linters. Use make install-linters first
install-linters                Install linters
format                         Formats the code. Must have goimports installed (use make install-linters).
dep                            Sorts dependencies
host-apps                      Build apps binaries 
bin                            Build `skywire-node`, `skywire-cli`, `manager-node`, `therealssh-cli`
docker-image                   Build docker image `skywire-runner`
docker-clean                   Clean docker system: remove container ${DOCKER_NODE} and network ${DOCKER_NETWORK}
docker-network                 Create docker network ${DOCKER_NETWORK}
docker-apps                    Build apps binaries for dockerized skywire-node. `go build` with  ${DOCKER_OPTS}
docker-bin                     Build `skywire-node`, `skywire-cli`, `manager-node`, `therealssh-cli`. `go build` with  ${DOCKER_OPTS}
docker-volume                  Prepare docker volume for dockerized skywire-node
docker-run                     Run dockerized skywire-node ${DOCKER_NODE} in image ${DOCKER_IMAGE} with network ${DOCKER_NETWORK}
docker-stop                    Stop running dockerized skywire-node ${DOCKER_NODE}
```

File: Makefile [hunk 1]
```diff
@@ -31,16 +49,14 @@ format: ## Formats the code. Must have goimports installed (use make install-lin
...
 
-test: ## Run tests for net
-	${OPTS} go test -race -tags no_ci -cover -timeout=5m ./internal/...
-	${OPTS} go test -race -tags no_ci -cover -timeout=5m ./pkg/...
+
 
 
 # Apps 
-host-apps: 
+host-apps: ## Build apps binaries
```
Comment (evanlinjin) [inline Makefile]: Change to `## Build app binaries`

**Review by evanlinjin — APPROVED** [review]
Perfect. Just change the gramatical error.

File: .travis.yml [hunk 2]
```diff
@@ -17,5 +18,4 @@ install:
   - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $GOPATH/bin v1.13.2
 
 script:
-  - make test
```
Comment (evanlinjin) [inline .travis.yml]: We need to run tests. Please keep this.

File: Makefile [hunk 3]
```diff
@@ -31,16 +49,14 @@ format: ## Formats the code. Must have goimports installed (use make install-lin
 	${OPTS} goimports -w -local github.com/skycoin/skywire ./cmd
 	${OPTS} goimports -w -local github.com/skycoin/skywire ./internal
 
-dep: ## sorts dependencies
+dep: ## Sorts dependencies
 	${OPTS} go mod vendor -v
 
-test: ## Run tests for net
```
Comment (evanlinjin) [inline Makefile:37]: We need to run go tests. Where are they now?

**Review by evanlinjin — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (ayuryshev) [reply Makefile]: Done

File: Makefile (hunk 3 above)
Comment (ayuryshev) [reply Makefile:37]: Returned:
```
test: ## Run tests for net
        ${OPTS} go test -race -tags no_ci -cover -timeout=5m ./internal/...
        ${OPTS} go test -race -tags no_ci -cover -timeout=5m ./pkg/...
```

**Review by evanlinjin — APPROVED** [review]

**Pull request merged**